arm-software/embedded/scripts/build_llvmlibc_overlay.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# A bash script to build the llvmlibc overlay for the Arm Toolchain for Embedded
+
+# The script creates a build of the toolchain in the 'build_llvmlibc_overlay'
+# directory, inside the repository tree.
+
+set -ex
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+REPO_ROOT=$( git -C ${SCRIPT_DIR} rev-parse --show-toplevel )
+BUILD_DIR=${REPO_ROOT}/build_llvmlibc_overlay
+
+clang --version
```

Comment:
What's this for?

Reply:
This looks like a leftover line from the early days of the original build.sh script. I'll remove it from these new scripts here and create a separate PR to remove it from build.sh.